core/ts/__tests__/utils.ts

```diff
@@ -145,7 +149,5 @@ export class TestHarness {
    * @param user - The keypair of the user.
    * @returns The state index of the user.
    */
-  getStateIndex = (user: Keypair): number => {
-    return this.stateIndices.get(user)!;
-  };
-}
\ No newline at end of file
+  getStateIndex = (user: Keypair): number => this.stateIndices.get(user)!;
+}
```

Comment:
we should decide whether to keep this one or use the other in `core/ts/__tests__/utils/MaciTestHarness.ts`

Reply:
oh right my bad I forgot to remove one of them, good catch. Kept inside test utils

Reply:
> oh right my bad I forgot to remove one of them, good catch. Kept inside test utils

cool, looks good to me!